keadm/app/cmd/util/k8sinstaller.go

```diff
@@ -49,7 +49,8 @@ func (ks *K8SInstTool) InstallTools() error {
 	case types.VersionNAInRepo:
 		return fmt.Errorf("Expected %s version is not available in OS repo", component)
 	case types.AlreadySameVersionExist:
-		return fmt.Errorf("Same version of %s already installed in this host", component)
+		fmt.Println("Same version of %s already installed in this host", component)
```

Comment:
Same version of %s  *is* already installed .
similarly at other places .

Reply:
do you mean to improve the message? i feel it's a separate change.

Reply:
ok , we can have a separate PR for this.